src/connectedk8s/HISTORY.rst

```diff
@@ -2,9 +2,13 @@
 
 Release History
 ===============
-1.6.0
```

Comment:
@TheOnlyWei , can you share why you are making this change?  Can you update the description with this explanation and link to the ADO Task tracking this work?

Reply:
Replied in Teams chat with details.

Reply:
Lets use the PR for discussion @TheOnlyWei for why the change is needed

Reply:
@bganapa I gave Narayan internal links in Teams Chat, so didn't make sense to link them in a public discussion.

But the reason I am making this change, is because AAD graph is deprecated.